src/rma/js/components/destruction-list/destruction-form.js

```diff
@@ -0,0 +1,134 @@
+import React, {useEffect, useState} from "react";
+
+import { SelectMultipleInput } from "./select";
+import { DateInput } from "./inputs";
+import { ZakenTable } from "./zaken-table";
+import { CreateModal} from "./create-modal";
+import { countObjectKeys } from "../../utils";
+
+
+function getFullUrl(url, filters) {
+    const query = Object.keys(filters).filter(k=>filters[k]).map(k => `${k}=${filters[k]}`).join('&');
```

Comment:
not a fan of this - won't use it to block the PR, but URL-encoding etc. is definitely relevant.

Using consumerjs should help with this though, for next iteration.